Source/Route.swift

```diff
@@ -76,7 +72,7 @@ public class Route {
 
             // url params should be unique
             if urlParamKeys.contains(name) {
-                let e: NSException = NSException(name:"Identical route params",
+                let e = NSException(name:"Identical route params",
```

Comment:
We should probably be using a swift error here again, unless there is a specific reason for using NSException?